src/AirshipBindings.iOS.Core/ApiDefinitions.cs

```diff
@@ -260,6 +260,12 @@ interface UAActivityViewController : IUIPopoverControllerDelegate, IUIPopoverPre
 		IntPtr Constructor(NSObject[] activityItems, [NullAllowed] UIActivity[] applicationActivities);
 	}
 
+    // @interface UABlockAction : NSObject <UAAction>
+    [BaseType(typeof(NSObject))]
+    interface UABlockAction : IUAAction
```

Comment:
Not sure if this is enough for it to  work, i think you need a constructor as well

Reply:
You should double check you can use it like a normal block action